frontend/views/containers/proposals/Mincome.vue

```diff
@@ -123,7 +123,7 @@ export default {
           this.$refs.formMsg.danger(L('Failed to change mincome. {codeError}', { codeError: e.message }))
           this.ephemeral.currentStep = 0
         }
-
+        done()
```

Comment:
Same concerns as with the other callback function in Contributions.vue (see comment below)

Reply:
`done()` is used to let the parent know when the submit is finished and set back `isSubmitting` to false. In this case (and the other 3 proposals), it reports to `ProposalTemplate` L178-180.

If we remove `done()` after the `submit` completes the button will keep showing the loading state. This isn't noticeable if the submit is successful because the modal closes, but if it fails, that's a problem.

Reply:
We can remove `done()` now with `ButtonSubmit`?

Reply:
It was removed yes